File: Corte1/Python/Ejercicio2/Ejercicio2.py
import math

class Circulo:
    def __init__(self, radio):
        self.radio = radio

    def calcular_area(self):
        return math.pi * self.radio ** 2


c1 = Circulo(5)
c2 = Circulo(3)

print(c1.calcular_area())
print(c2.calcular_area())

File: Corte1/Java/Ejercicio2/Ejercicio2.java
class Circulo {
    double radio;

    public Circulo(double radio) {
        this.radio = radio;
    }

    public double calcularArea() {
        return Math.PI * radio * radio;
    }
}

public class Main {
    public static void main(String[] args) {
        Circulo c1 = new Circulo(5);
        Circulo c2 = new Circulo(3);

        System.out.println(c1.calcularArea());
        System.out.println(c2.calcularArea());
    }
}
